superflore/utils.py

```diff
@@ -74,13 +74,12 @@ def load_pr():
         err('Failed to open PR title/message file!')
         err(
             'Please supply the %s and %s files' % (
-                '.pr_message.tmp',
-                '.pr_title.tmp'
+                '.pr-message.tmp',
+                '.pr-title.tmp'
```

Comment:
Why change this?

Reply:
It was a typo, so replacing '_' by '-' as in the file path.